data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
+			`"`
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."
+			"I feel like the only Hai on this planet who regrets joining the Unfettered."
+			"I want to go home."
+		word
+			`"`
+news "unfettered hai after wanderers invasion"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"We shall conquer the worlds to the north as is our right. It is inevitable that the strong shall defeat the weak."
+			"I have fought with the stupid birds to the north. They have no knowledge of proper battle. Our war with them will be swift."
+			"Those overgrown birds have fantastic weaponry, yet they are too cowardly to use it."
+			"I wonder why the birds do not simply fight back. Their weapons are clearly capable of doing so."
+			"I wish I could get my hands on some of that Wanderer tech."
+			"Wanderer worlds are pristine! We have won a great prize in our battle against the stupid birds."
+			"The true Hai will no longer be confined in their Drak-imposed cage."
+			"I'll see great glory in fighting the weak-willed birds."
+			"The worlds up north are nothing like the ones we were confined to. On the Wanderer worlds, the air is clean and the water is as clear as glass."
+		word
+			`"`
+news "unfettered hai after wanderers invasion regret"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"I had to shoot down a Wanderer ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
+			"The Wanderers hardly try to fight back. It makes me wonder if they want us to conquer their planets."
+			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
+			"I came here to do battle with warriors, not with merchants and cowardly birds."
+			"What challenge is there in killing the weak? I desire to battle with those who are worthy."
+			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and threaten to kill me."
+			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai langauge. If it were the other way around, I'd be dead."
+			"I was held prisoner on a Wanderer world. They said that we were welcome to take their planets, if it was their fate. It must be a trap."
+			"I was held prisoner on a Wanderer world. Even though I attacked them, they were so kind to me. I don't want to fight them anymore, I'd feel too guilty."
```

Comment:
The Wanderers do not appear to have taken prisoners in the narrative. Also, birds.

Reply:
I was basing it off of this quote from Iktak Rek: `Recently we managed to [take, capture] some of them as prisoners," he says.`, when asked by the player how the Wanderers learned the Hai language.

Reply:
Huh, I must've totally missed that in my reading. Damn

Reply:
You don't mind if I resolve this, right?

Reply:
So it's worth noting that they never identify themselves as the Wanderers at the start, the player can see the naming of them in the game, and so the game simply assumes that the player knows they're "Wanderers" and treats you accordingly. Though amusingly, the mission where they refer to having captured prisoners is the first time they refer to themselves as Wanderers and it's not just meta knowledge.
I'm going to say yes you can resolve this, but I think it would be interesting to include a news item or two about the use of the name. It seems to me that the Unfettered must have been raiding them for a while, and have established a habit of referring to them in some manner before they ever knew they were "Wanderers".
Perhaps there could be some entries implying a split, those who are still bent on invading merely referring to them as birds, and those who have developed some sympathy using the less popular "Wanderers"?

Reply:
I'm going to assume that the Wanderers call themselves that, or whatever is the equivalent in their local language. I added a couple of news lines that referred to this.

Reply:
I like them.